library/templates/dcla_culturalinstitutions.yml

```diff
@@ -0,0 +1,34 @@
+dataset:
+  name: &name dcla_culturalinstitutions
+  version: "{{ version }}"
+  acl: public-read
+  source:
+    socrata:
+      uid: u35m-9t32
+      format: csv    
+    options:
+      - "AUTODETECT_TYPE=NO"
+      - "EMPTY_STRING_AS_NULL=YES" 
+      - "X_POSSIBLE_NAMES=longitude,Longitude,Lon,lon,x"
+      - "Y_POSSIBLE_NAMES=latitude,Latitude,Lat,lat,y"
+    geometry:
+      SRS: EPSG:2263
```

Comment:
based on what I'm seeing, this should be EPSG: 4326

Reply:
The srcSRS value in the recipes app says EPSG:2263. Should I change it anyways?

Reply:
@SPTKL does it make a difference if `type: NONE`?

Reply:
But this dataset does have a point geometry, and it comes in longitude and latitude, it would be miss leading to use 2263

Reply:
Got it. I switched it to EPSG:4326!